artifactory/cli.go

```diff
@@ -621,15 +621,98 @@ func GetCommands() []cli.Command {
 	}
 }
 
-func getGlobalConfigFlag() []cli.Flag {
+func getBaseBuildToolsConfigFlags() []cli.Flag {
 	return []cli.Flag{
 		cli.BoolFlag{
 			Name:  "global",
-			Usage: "[Default: false] Set to true, if you'd like to configuration to be global (for all projects). Specific projects can override the global configuration.` `",
+			Usage: "[Default: false] Set to true if you'd like the configuration to be global (for all projects). Specific projects can override the global configuration.` `",
+		},
+		cli.BoolTFlag{
+			Name:  "interactive",
+			Usage: "[Default: true] Set to false if you do not want the config command to be interactive.` `",
+		},
+		cli.BoolTFlag{
+			Name:  "resolveFromArtifactory",
+			Usage: "[Default: true] Set to false if you do not wish to resolve dependencies from Artifactory.` `",
+		},
+		cli.StringFlag{
+			Name:  "resolutionServerId",
+			Usage: "[Optional] Artifactory server ID for resolution.` `",
+		},
+		cli.BoolTFlag{
+			Name:  "deployToArtifactory",
+			Usage: "[Default: true] Set to false if you do not wish to deploy artifacts to Artifactory.` `",
+		},
+		cli.StringFlag{
+			Name:  "deploymentServerId",
+			Usage: "[Optional] Artifactory server ID for deployment.` `",
```

Comment:
Artifactory server ID for deployment. --> Artifactory server ID for deployment. The server should configured using the 'jfrog rt c' command.